search-sentiments-map/public/scripts/map-script.js

```diff
@@ -60,16 +60,25 @@ function initMap() {
 function loadMapOutline() {
   // Load country data after finished loading in geojson.
   map.data.loadGeoJson('countries.geojson', null, function () {
-    loadCountryData();
+    setNewTrend();
   });
 }
 
 /** Loads the country sentiment score from Datastore. */
 function loadCountryData() {
-  map.data.forEach(function(row) {
-    // Currently a random value, will be changed to call sentiment value.
-    // TODO(ntarn): Add the sentiment value for the country. 
-    const dataVariable = Math.floor(Math.random() * Math.floor(100));
+  map.data.forEach(function (row) {
+    const countryCode = row.getId();
+    let topicData = getCurrentCustomSearchData();
+    let countryData = topicData.countries
+      .filter(countries => countries.country === countryCode);
+    let dataVariable = 0;
+    if (countryData.length != 0) {
+      dataVariable = countryData[0].averageSentiment;
+      // TODO(ntarn): Remove console.log statements when finished debugging.
+      console.log('ntarn debug loadCountryData: ' + 'country: ' + countryData[0].country + ' averageSentiment: ' + dataVariable);
```

Comment:
remove the console log

Reply:
I need it for the sentiment chart currently.

Reply:
Is this line too long?